x-pack/test/functional_with_es_ssl/apps/cases/list_view.ts

```diff
@@ -329,6 +338,30 @@ export default ({ getPageObject, getService }: FtrProviderContext) => {
         await cases.casesTable.validateCasesTableHasNthRows(4);
       });
 
+      it('filters cases from the list using an id search', async () => {
+        await testSubjects.missingOrFail('cases-table-loading', { timeout: 5000 });
+
+        const input = await testSubjects.find('search-cases');
+        await input.type(caseIds[0]);
+        await input.pressKeys(browser.keys.ENTER);
+
+        await cases.casesTable.validateCasesTableHasNthRows(1);
```

Comment:
I think we should check if the expected case shown in the results after checking for the total number of cases. You can do that by `await find.allByCssSelector('[data-test-subj*="cases-table-row-${caseId}"');`. There is already a function called `getCaseFromTable` in `x-pack/test/functional/services/cases/list.ts` but it returns the case by index (position in the table). Maybe you can rename this to `getCaseByIndex` and create another one called `getCaseById`.

Reply:
done 👍